searchlib/src/vespa/searchlib/query/streaming/wand_term.cpp

```diff
@@ -34,8 +34,9 @@ WandTerm::evaluate() const
 }
 
 void
-WandTerm::unpack_match_data(uint32_t docid, const ITermData& td, MatchData& match_data)
+WandTerm::unpack_match_data(uint32_t docid, const ITermData& td, MatchData& match_data, const fef::IIndexEnvironment& index_env)
 {
+    (void) index_env;
```

Comment:
When the type is specific you do not need a name for the parameter, and then you do not need the explicit (void).